### ISSUE TITLE
FieldOptions not support repeated field

### ISSUE THREAD

**Issue by bafflingbug** [devkanro/intellij-protobuf-plugin#260]
### Reproducible example
```protobuf
// validate.proto
import "google/protobuf/descriptor.proto";

extend google.protobuf.FieldOptions {
  optional FieldRules rules = 1001;
}

message FieldRules {
  oneof type {
    StringRules   string   = 1;
  }
}

message StringRules {
  repeated string in     = 10;
}
```

```protobuf
import "validate.proto"

message Request {
  string option = 1 [(validate.rules).string = {in: ["add", "del", "update"]}];
}
```

there is an error
![image](https://github.com/devkanro/intellij-protobuf-plugin/assets/23716431/147543a1-dcc5-4b33-b149-abb44ccd4d99)

### in the official plugin
![image](https://github.com/devkanro/intellij-protobuf-plugin/assets/23716431/b77b1eec-6d2d-4493-8f2d-e594f6899920)

**Comment by bafflingbug** [comment]
GoLand 2024.1.4
Build #GO-241.18034.61, built on June 21, 2024

Protobuf plugin: 2.0.0

**Comment by devkanro** [comment]
It will be resolved in 2.0.1
